README.md

```diff
@@ -46,7 +46,7 @@ The deployed version of this application uses a remote database.
 Public read-only access is available with the following configuration:
 
 ```yaml
-name: dj_hetmech
+name: connectivity_db
```

Comment:
should we also change the local docker to use this database name?

`--env POSTGRES_DB=dj_hetmech` below? And what about `--env POSTGRES_USER=dj_hetmech`?

Reply:
I only renamed the database to `connectivity_db` to avoid confusion. The database user name is still `dj_hetmech`. I will modify the docker command in `README.md` tomorrow.